<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComfortQuestionnaireOptions</class>
 <widget class="QWidget" name="ComfortQuestionnaireOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Questionnaire options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>settingLabel</zorder>
   <zorder>back</zorder>
  </widget>
  <widget class="QCheckBox" name="enableQuestionnaire">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>45</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Questionnaire enabled</string>
   </property>
  </widget>
  <widget class="QWidget" name="options" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>300</width>
     <height>160</height>
    </rect>
   </property>
   <widget class="QPushButton" name="startNow">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start now</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Question Time 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="numberOfQuestionnaires" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>25</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QPushButton" name="lower">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="number">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>0</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>15.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="raise">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">+</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="changeTime1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QPushButton" name="lowerTime1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="raiseTime1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">+</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>6</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumTime">
      <time>
       <hour>21</hour>
       <minute>59</minute>
       <second>59</second>
      </time>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="changeTime2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QPushButton" name="lowerTime2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="raiseTime2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">+</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumTime">
      <time>
       <hour>21</hour>
       <minute>59</minute>
       <second>59</second>
      </time>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelNumberOfQuestionnaires">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>How many times per day?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="changeTime3" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QPushButton" name="lowerTime3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="raiseTime3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">+</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>19</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumTime">
      <time>
       <hour>21</hour>
       <minute>59</minute>
       <second>59</second>
      </time>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelTime3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Question Time 3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Question Time 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>45</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>25</y>
      <width>45</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_right_white.svg</normaloff>:/settings/icons/arrow_right_white.svg</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
